internal/services/order/order.go

```diff
@@ -1,13 +1,219 @@
 package order
 
-func BuyOrder(BReq BuyRequest) (bRes BuyResponse, err error) {
-	return bRes, err
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/services/stock_exchange"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/google/uuid"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type result struct {
+	StockName     string
+	TotalQuantity int
 }
 
-func SellOrder(BReq SellRequest) (sRes SellResponse, err error) {
-	return sRes, err
+func BuyOrder(bReq BuyRequest) (bRes stock_exchange.OrderResponse, err error) {
+	var stocks model.StocksFeed
+	var account model.TradingAccount
+
+	if bReq.BookType == market {
+		if err = config.DB.Table("stocks_feed").Where("stock_name=?", bReq.StockName).Last(&stocks).Error; err != nil {
+			return bRes, err
```

Comment:
write descriptive error message all over the code.

Reply:
done